developer/src/tike/xml/help/contexthelp.xml

```diff
@@ -331,33 +331,185 @@
       <p>The debugger can be used without the debug information by clicking on Test without debugger.</p>
     </Control>
   </Form>
-
+  
+  <!-- New file details -->
   <Form Name="context/new-file-details">
     <Control Name="*" Title="New File">
       <p>Enter the name for your new keyboard. Click on the Browse button to change the location of the new keyboard.</p>
     </Control>
   </Form>
-
+  
+  <!-- Package Editor -->
   <Form Name="context/package-editor">
-    <Control Name="cbImageFile" Title="Image File">
-      <p>You can include an image file that will be displayed to the left of the install details when the package is installed. This image should be 140 pixels wide and 250 pixels high.</p>
+    <Control Name="pages" Title="Keyboard Package">
+      <p>A keyboard package is most likely to have 7 tabs:
+        <ul>
+          <li>Files</li>
+          <li>Keyboards</li>
+          <li>Lexical Models</li>
+          <li>Details</li>
+          <li>Shortcuts</li>
+          <li>Source</li>
+          <li>Build</li>
+        </ul>
```

Comment:
I don't think that this is useful documentation. Does this come from help.keyman.com?

Reply:
It is not on help.keyman and it is triggered when clicking on a tab the second time:
![image](https://github.com/keymanapp/keyman/assets/90595388/65a55f76-3084-49d5-987d-aefcf121b7ae)
Should I provide a brief description for each tab?